src/main/java/uk/gov/companieshouse/charges/data/service/ChargesService.java

```diff
@@ -45,14 +37,19 @@ public InternalApiClient getInternalApiClient() {
      */
     public void upsertCharges(String companyNumber, String chargeId,
```

Comment:
I noticed that this method is not transactional and also the method in the controller where it's called. If I understood  properly we have a problem on local and rebel1 databases where transactional is not supported. In the Insolvency Data API project, two services classes have been created. So, not sure which approach we want to use here. @zahidm3 @adeshpandech Please let me know your thoughts.

Reply:
I will mark this as resolved. As we discussed today in SU meeting we are going to handle the transactional in a separate story. Thanks.